src/graphql/root/mutation/account-update-default-wallet-id.ts

```diff
@@ -0,0 +1,45 @@
+import { Accounts } from "@app"
+import { mapError } from "@graphql/error-map"
+import { GT } from "@graphql/index"
+
+import AccountUpdateDefaultWalletIdPayload from "@graphql/types/payload/account-update-default-wallet-id"
+import WalletId from "@graphql/types/scalar/wallet-id"
+
+const AccountUpdateDefaultWalletIdInput = GT.Input({
+  name: "AccountUpdateDefaultWalletIdInput",
+  fields: () => ({
+    walletId: { type: GT.NonNull(WalletId) },
+  }),
+})
+
+const AccountUpdateDefaultWalletIdMutation = GT.Field({
+  type: GT.NonNull(AccountUpdateDefaultWalletIdPayload),
+  args: {
+    input: { type: GT.NonNull(AccountUpdateDefaultWalletIdInput) },
+  },
+  resolve: async (_, args, { domainAccount }: { domainAccount: Account }) => {
+    const { walletId } = args.input
+
+    if (walletId instanceof Error) {
+      return { errors: [{ message: walletId.message }] }
+    }
+
+    const result = await Accounts.updateDefaultWalletId({
+      walletId,
+      accountId: domainAccount.id,
+    })
+
+    if (result instanceof Error) {
+      const appErr = mapError(result)
+      return { errors: [{ message: appErr.message || appErr.name }] }
```

Comment:
```suggestion
      return { errors: [mapError(result)] }
```

Reply:
mapError already do message/name coalescing
```js
const errorName = error.name as ApplicationErrorKey
let message = error.message || errorName || ""
```

Reply:
With the current destructuring I get:
```
{
    "data": {
        "accountUpdateDefaultWalletId": {
            "errors": [
                {
                    "message": "Unknown error occurred (code: InvalidWalletId)"
                }
            ],
            "account": null
        }
    }
}
```

And with the direct `mapError(result)` suggested I get:
```
{
    "errors": [
        {
            "message": "Unknown error occurred (code: InvalidWalletId)",
            "code": "UNKNOWN_CLIENT_ERROR"
        }
    ],
    "data": null
}
```